backend/api/pipeline.proto

```diff
@@ -75,6 +75,13 @@ service PipelineService {
     };
   }
 
+    // Finds a specific pipeline by Name (within a namespace).
+    rpc GetPipelineByName(GetPipelineByNameRequest) returns (Pipeline) {
```

Comment:
NIT: There is extra indention for this rpc block. Can you update it?

Reply:
done